docs/content/preview/tutorials/AI/ai-localai.md

```diff
@@ -0,0 +1,364 @@
+    yugabytedb/yugabyte:{{< yb-version version="preview" format="build">}} \
+    bin/yugabyted start \
+    --base_dir=/home/yugabyte/yb_data --background=false
+
+docker run -d --name yugabytedb-node2 --net custom-network \
+    -p 15434:15433 -p 7002:7000 -p 9002:9000 -p 5434:5433 \
+    -v ~/yb_docker_data/node2:/home/yugabyte/yb_data --restart unless-stopped \
+    yugabytedb/yugabyte:{{< yb-version version="preview" format="build">}} \
+    bin/yugabyted start --join=yugabytedb-node1 \
+    --base_dir=/home/yugabyte/yb_data --background=false
+
+docker run -d --name yugabytedb-node3 --net custom-network \
+    -p 15435:15433 -p 7003:7000 -p 9003:9000 -p 5435:5433 \
+    -v ~/yb_docker_data/node3:/home/yugabyte/yb_data --restart unless-stopped \
+    yugabytedb/yugabyte:{{< yb-version version="preview" format="build">}} \
+    bin/yugabyted start --join=yugabytedb-node1 \
+    --base_dir=/home/yugabyte/yb_data --background=false
+```
+
+The database connectivity settings are provided in the `{project_dir}/.env` file and do not need to be changed if you started the cluster with the preceding command.
+
+Navigate to the YugabyteDB UI to confirm that the database is up and running, at <http://127.0.0.1:15433>.
+
+## Get started with LocalAI
+
+Running [LocalAI](https://localai.io/basics/getting_started/) on your machine, or on a virtual machine, requires sufficient hardware. Be sure to check the specifications before installation. This application uses Docker to run a BERT text-embedding model.
+
+Execute the following command to run this model in Docker:
+
+```sh
+docker run -ti -p 8080:8080 localai/localai:v2.8.0-ffmpeg-core bert-cpp
+```
+
+The following output is generated, exposing a REST endpoint to interact with the LLM.
+
+```output
+...
+You can test this model with curl like this:
+
+curl http://localhost:8080/embeddings -X POST -H "Content-Type: application/json" -d '{
+  "input": "Your text string goes here",
+  "model": "bert-cpp-minilm-v6"
+}'
+```
+
+## Load the schema and seed data
+
+This application requires a database table with information about popular programming languages. This schema includes a `programming_languages` table.
+
+1. Copy the schema to the first node's Docker container.
+
+    ```sh
+    docker cp {project_dir}/sql/schema.sql yugabytedb-node1:/home
+    ```
+
+1. Copy the seed data file to the Docker container.
+
+    ```sh
+    docker cp {project_dir}/sql/data.sql yugabytedb-node1:/home
+    ```
+
+1. Execute the SQL files against the database.
+
+    ```sh
+    docker exec -it yugabytedb-node1 bin/ysqlsh -h yugabytedb-node1 -f /home/schema.sql
+    docker exec -it yugabytedb-node1 bin/ysqlsh -h yugabytedb-node1 -f /home/data.sql
+    ```
+
+## Start the application
+
+This command-line application takes an input in natural language and returns a response from YugabyteDB. By converting text to embeddings, a similarity search is executed using `pgvector`.
```

Comment:
The LLM part is missed in this chain

`This command-line application takes an input in natural language and returns a response from YugabyteDB.`

Let's add the "uses a locally running LLM to produce an embedding for the user prompt..."